CK3ToEU4/Source/Configuration/Configuration.cpp

```diff
@@ -136,45 +136,45 @@ void Configuration::registerKeys()
 void Configuration::verifyCK3Path()
 {
 	if (!commonItems::DoesFolderExist(CK3Path))
-		throw std::runtime_error(CK3Path + " does not exist!");
+		throw std::runtime_error(CK3Path.string() + " does not exist!");
 	// TODO: OSX and Linux paths are speculative
-	if (!commonItems::DoesFileExist(CK3Path + "/binaries/ck3.exe") && !commonItems::DoesFileExist(CK3Path + "/CK3game") &&
-		 !commonItems::DoesFileExist(CK3Path + "/binaries/ck3"))
-		throw std::runtime_error(CK3Path + " does not contain Crusader Kings 3!");
-	if (!commonItems::DoesFileExist(CK3Path + "/game/map_data/positions.txt"))
-		throw std::runtime_error(CK3Path + " does not appear to be a valid CK3 install!");
-	Log(LogLevel::Info) << "\tCK3 install path is " << CK3Path;
+	if (!commonItems::DoesFileExist(CK3Path / "binaries/ck3.exe") && !commonItems::DoesFileExist(CK3Path / "CK3game") &&
+		 !commonItems::DoesFileExist(CK3Path / "binaries/ck3"))
+		throw std::runtime_error(CK3Path.string() + " does not contain Crusader Kings 3!");
+	if (!commonItems::DoesFileExist(CK3Path / "game/map_data/positions.txt"))
+		throw std::runtime_error(CK3Path.string() + " does not appear to be a valid CK3 install!");
+	Log(LogLevel::Info) << "\tCK3 install path is " << CK3Path.string();
 	CK3Path += "/game/"; // We're adding "/game/" since all we ever need from now on is in that subdirectory.
```

Comment:
I do not think += works for paths as it does for strings. Please check this.

Reply:
I had to do something like
CK3Path = CK3Path / "game";
on CK2toEU4 since += didn't work at all. Just saying need to test if it's ok at all. Hope I am wrong.